routes/users.js

```diff
@@ -1,20 +1,19 @@
 const usersRouter = require("express").Router();
-const { createUser, getUser, getUserByUsername, getOrdersByUser, patchUser, deleteUser } = require("../db");
+const { getAllUsers, createUser, getUser, getUserByUsername, getOrdersByUser, patchUser, deleteUser } = require("../db");
 const { isLoggedIn, isAdmin } = require("./util")
 const jwt = require("jsonwebtoken");
 require('dotenv').config();
 const { JWT_SECRET = 'neverTell' } = process.env;
 
-// usersRouter.get("/", async (req, res, next) =>{
-//     try {
-//         const allUsers = await getAllUsers();
-//         res.send(allUsers)
-//     } catch (error) {
-//         throw error
-//     }
-// } );
+usersRouter.get("/", isAdmin, async (req, res, next) =>{
+    try {
+        const allUsers = await getAllUsers();
+        res.send(allUsers)
+    } catch (error) {
+        throw error
+    }
+} );
 
-// POST /api/users/register
```

Comment:
Don't have to add them back, but these comments can actually be beneficial for immediately knowing the route to manually tests a handler, instead of having to figure out where users.js is required, and then where that is required to determine what comes before `/register` if I want to try calling this thing from the client or via curl or postman, etc.